amplify/src/components/CheckSpan.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const ShowHideSpan = (props, style) => {
+  return (
+  <span className={props.hidden ? style.hidden : style.show} children={'✔'}>
```

Comment:
This is the ternary I mentioned. I'm just passing in 'style' to get it to stop yelling but I know that shouldn't work. Do I need to import style in some way?.